karma.conf.js

```diff
@@ -0,0 +1,57 @@
+module.exports = function (config) {
+    config.set({
+
+        browsers: ['PhantomJS'],
+
+        frameworks: ['jasmine'],
+
+        files: ['node_modules/babel-core/browser-polyfill.js', {
+            pattern: './src/tests.webpack.js', watched: false
+        }],
+
+        preprocessors: {
+            'src/tests.webpack.js': ['webpack']
+        },
+
+        webpack: {
```

Comment:
This adds another level of webpack configuration; Should this align with the webpack configuration currently in master ? ie https://github.com/onefinestay/react-daterange-picker/blob/v0.12.1/gulpfile.js#L41